benchmarks/operator_benchmark/pt/channel_shuffle_test.py

```diff
@@ -39,7 +39,7 @@ class ChannelSHuffleBenchmark(op_bench.TorchBenchmarkBase):
     def init(self, batch_size, channels_per_group, height, width, groups):
         channels = channels_per_group * groups
         data_shape = (batch_size, channels, height, width)
-        self.input_data = torch.rand(data_shape)
+        self.input_data = torch.rand(data_shape).contiguous(memory_format=torch.channels_last)
```

Comment:
Let's benchmark both formats?

Reply:
Do you mean adding both formats for both diffs or just the top diff?